spec/models/mentorship_spec.rb

```diff
@@ -13,21 +18,22 @@
     it "Should choose a valid person to mentor" do
       user = create :mentor
       mentee = create :mentee
+      miu = create :mentor_industry_user, user: user, mentor_industry: MentorIndustry.where(name: "Technology").first
```

Comment:
This does not need to be saved as a variable, if it's not being used later (delete `miu = `), also below